ndoh_hub/settings.py

```diff
@@ -51,8 +51,13 @@
     "django.contrib.staticfiles",
     # 3rd party
     "raven.contrib.django.raven_compat",
-    "rest_framework",
     "rest_framework.authtoken",
+    "rest_auth",
```

Comment:
Are all these auth things needed? Can't we do token auth?

Reply:
Token should be fine. I'll remove rest_auth and the url